fasta/pFasta/pFasta.go

```diff
@@ -145,3 +145,44 @@ func Read(inFile string) []PFasta {
 	exception.PanicOnErr(err)
 	return records
 }
+
+// ToMap converts a slice of pFasta records (e.g. the output of the Read function)
+// to a map of sequences keyed to the sequences name.
+func ToMap(ref []PFasta) map[string][]pDna.Float32Base {
+	m := make(map[string][]pDna.Float32Base)
+	for i := range ref {
+		_, ok := m[ref[i].Name]
+		if !ok {
+			m[ref[i].Name] = ref[i].Seq
+		} else {
+			log.Panicf("%s used for multiple pFasta records. record names must be unique.", ref[i].Name)
+		}
+	}
+	return m
+}
+
+/*
+// QC tests if each base in a single pFasta sequence is valid. Valid means either gap {0 0 0 0} or non-gap (bases' probabilities add up to 1)
```

Comment:
Can commented code be removed?

Reply:
I'm keeping the code for now because it helped me debug and I think it's useful to have QC and MakeValid functions for the future in the cases where we get pFastas as inputs. For now our functions generate pFastas so we already make sure they are valid.

Reply:
May remove this code